src/components/confirm/confirm.stories.tsx

```diff
@@ -0,0 +1,200 @@
+import React, { useState } from "react";
+
+import Confirm from ".";
+import Button from "../button";
+import isChromatic from "../../../.storybook/isChromatic";
+
+const isOpenForChromatic = isChromatic();
```

Comment:
could you rename `isOpenForChromatic` to `defaultOpenState`, so it would not confuse users?